slingr-stack/slack-package
# #17494 review slack package

Pull-request for issue #17505 created by @germanm98

## What it does?

Review the Slack package, added need listeners

## How to test it?
Create a slack app at https://api.slack.com/apps
Add the package to a existing app and set it up.
In your Slack app, subscribe your bot to some slack **events**,
set and **interactive message** and also a **slash command**.
Check that all the webhooks mentioned above arrives to your app.


## Technical notes

There are no technical notes.

## Deployment notes

There are no deployment notes.
